explorer/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Unless a later match takes precedence, @akegalj will be requested
```

Comment:
Does it work as expected? For example does `src` mean `explorer/src`? Is this file considered at all (given that it's no longer in the root)?